src/FlightController.cpp

```diff
@@ -112,88 +299,51 @@ bool FlightController::setRc(const uint16_t roll, const uint16_t pitch,
                              const uint16_t aux3, const uint16_t aux4,
                              const std::vector<uint16_t> auxs)
 {
-    if(isFirmwareMultiWii() && hasDynBal()) {
```

Comment:
Where did this go?

Reply:
I will need to look at this again

Reply:
It's not a high priority, but is the original 8bit Arduino MultiWii firmware still supported?